pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -120,8 +119,16 @@ type PodAutoscalerList struct {
 	Items []PodAutoscaler `json:"items"`
 }
 
-func (kpa *PodAutoscaler) scaleBoundInt32(key string) int32 {
-	if s, ok := kpa.Annotations[key]; ok {
+func (pa *PodAutoscaler) Class() string {
+	if c, ok := pa.Annotations[autoscaling.ClassAnnotationKey]; ok {
+		return c
+	}
+	// Default to "pa" class for backward compatibility.
+	return autoscaling.KPA
+}
+
+func (pa *PodAutoscaler) scaleBoundInt32(key string) int32 {
```

Comment:
Not necessarily in this PR, but stumbled over it: Should this be named "annotationAsInt32" or something similar? It's not specific to scaleBounds.